certified-connectors/CRMBot/readme.md

```diff
@@ -0,0 +1,12 @@
+Create powerful conversation experience for your customers right from your CRM system. Give users new ways to interact with your company by building engaging voice and text-based conversational interfaces, such as voice apps and chatbots, powered by AI. Connect with users on your website, mobile app, the Google Assistant, Amazon Alexa, Facebook Messenger, and other popular platforms and devices. And do it directly from your existing CRM implementation. 
+
+## Prerequisites
+
+Our clients have to undergo a registration process (currently offered through MS Teams chatbot as well as our website/sales bot. This provides them with a secret API key which is necessary to use our connector.
```

Comment:
There appears to be a missing ending parenthesis here for the opening one right before "...currently offered through"

Reply:
Also resolved cc @dougelliott